### PR TITLE
Feature/ver 84

### PR DESCRIPTION
**Pull request by Ellenn-A**
- removed packages @aries-framework/indy-vdr, @hyperledger/indy-vdr-nodejs and @hyperledger/indy-vdr-shared from veritable-cloudagent
- updated Agent instantiation, configuration and command line variables to reflect the removal of indy
- update references to did:sov replacing them with `https://didcomm.org/out-of-band/1.1/invitation`
- updated tests so they still pass as expected
- removed all references to the word indy
- deleted extra readme and updated + renamed remaining readme README_ARIES.md ->  README_VERITABLE.md
- deleted scripts and files related to using indy
- deleted Docker file and replaced it by afj-arm.dockerfile
- bumped version to 0.3.0
- from cliConfig.json removed: ` "indyLedger": [{...}]`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,34 +0,0 @@
-FROM ubuntu:18.04 as base
```
Comment (mattdean-digicatapult) [inline Dockerfile:1]: I understand the other dockerfile is what we want for now. Can we rename that to just `Dockerfile` replacing this file?

File: tests/credential.test.ts [hunk 2]
```diff
@@ -205,15 +205,20 @@ describe('CredentialController', () => {
...
+      //removed:
+      // issuerDid: 'WghBqNdoFjaYh6F5N9eBF',
+      // schemaId: 'WgWxqztrNooG92RXvxSTWv:2:test:1.0',
+      // schemaIssuerDid: 'WghBqNdoFjaYh6F5N9eBF',
+      // schemaName: 'test',
+      // schemaVersion: '1.0',
+
+      // autoAcceptCredential: 'always',
+      // comment: 'test',
+
```
Comment (mattdean-digicatapult) [inline tests/credential.test.ts:217]: remove comments

File: tests/credential.test.ts [hunk 3]
```diff
@@ -244,17 +247,31 @@ describe('CredentialController', () => {
...
+      //added proposal request - credentialFormars is required even if empty
+      const proposalRequest = {
+        credentialFormats: {},
+      }
 
-      const response = await request(app).post(`/credentials/${testCredential.id}/accept-proposal`)
+      const response = await request(app)
+        .post(`/credentials/${testCredential.id}/accept-proposal`)
+        .send(proposalRequest) //added proposal request
+      console.log('response' + response)
```
Comment (mattdean-digicatapult) [inline tests/credential.test.ts:258]: remove `console.log`s

File: tests/utils/helpers.ts [hunk 4]
```diff
@@ -82,13 +82,15 @@ export function getTestDidRecord() {
   return json
 }
 
+//removed from below:
+//  imageUrl: 'https://example.com/image-url',
+//  goal: 'string',
```
Comment (mattdean-digicatapult) [inline tests/utils/helpers.ts:87]: remove comment

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]
a few small changes

File: Dockerfile (hunk 1 above)
Comment (Ellenn-A) [reply Dockerfile:1]: done

File: tests/credential.test.ts (hunk 2 above)
Comment (Ellenn-A) [reply tests/credential.test.ts:217]: done

File: tests/credential.test.ts (hunk 3 above)
Comment (Ellenn-A) [reply tests/credential.test.ts:258]: done

File: tests/utils/helpers.ts (hunk 4 above)
Comment (Ellenn-A) [reply tests/utils/helpers.ts:87]: done

**Review by mattdean-digicatapult — APPROVED** [review]
lgtm

**Pull request merged**